src/phy/loopback.rs

```diff
@@ -14,7 +14,7 @@ use crate::phy::{self, Device, DeviceCapabilities};
 use crate::time::Instant;
 
 /// A loopback device.
-#[derive(Debug)]
+#[derive(Debug, Default)]
```

Comment:
I don't think this is desirable; end users should use `Loopback::new`. `Loopback` in this case behaves like an OS resource, not just an object.